docs/finops-alerts.md

```diff
@@ -54,7 +54,7 @@ style="background-color:#edf; border:solid 1px #609;">
 
 ## Deploy FinOps alerts
 
-FinOps alerts works best with the <strong>Owner</strong> role. For least privilege roles, contributor and user access admin roles are required.
+FinOps alerts works best with the <strong>Owner</strong> role. For least privilege roles, contributor and role based access control administrator roles are required.
```

Comment:
Minor formatting, but just to confirm, is full Contributor least priv or would something like Logic App contributor be acceptable?
```suggestion
FinOps alerts works best with the <strong>Owner</strong> role. For least privileged access, use the Contributor and Role Based Access Control Administrator roles.
```